GCRCatalogs/alphaq.py

```diff
@@ -26,6 +26,13 @@ def md5(fname, chunk_size=65536):
             hash_md5.update(chunk)
     return hash_md5.hexdigest()
 
+def _calc_conv(mag, shear1, shear2):
+    slct = mag < 0.2
```

Comment:
here you select mag < 0.2, but you use mag < 0.1 when modifying magnification itself. Just to make sure this is intended.

Reply:
Good catch!